### ISSUE TITLE
 Undefined control sequence

### ISSUE THREAD

**Issue by habedi** [tex-xet/xepersian#29]
Hi, 
Great work; Btw, When I try to use Xepersian package I confront this error:

```Bash

(/usr/share/texlive/texmf-dist/tex/xelatex/bidi/tc-xetex-bidi.def))
(/usr/share/texlive/texmf-dist/tex/xelatex/bidi/loadingorder-xetex-bidi.def))
****************************************************
* 
* xepersian package (Persian for LaTeX, using XeTeX engine)
* 
* Description: The package supports Persian
* typesetting, using fonts provided in the
* distribution.
* 
* Copyright (c) 2008--2017 Vafa Khalighi
* 
* v17.2, 2017/01/16
* 
* License: LaTeX Project Public License, version
* 1.3c or higher (your choice)
* 
* Location on CTAN: /macros/xetex/latex/xepersian
* 
* Issue tracker: https://github.com/tex-xet/xepersian/issues
* 
* Support: persian-tex@tug.org
****************************************************

(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/loadingorder-xepersian.def
) (/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/listings-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/algorithmic-xepersian.def)
 (/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/algorithm-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/footnote-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/framed-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/hyperref-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/tocloft-xepersian.def)
(/usr/share/texlive/texmf-dist/tex/xelatex/xepersian/book-xepersian.def)
! Undefined control sequence.
\setlatintextfont code ..._fontspec_pass_args:nnn 
                                                  \__xepersian_setlatintextf...
l.878 ...{lmromanslant10-bold}]{lmroman10-regular}
                                                  %
```

**Comment by vafa** [comment]
Update your TeX distribution.

**Comment by habedi** [comment]
It's not easy to update the whole Tex distribution just for a small bug. Btw, where can I download Xepersian 17.3?

**Comment by vafa** [comment]
There is no bug. Your `fontspec` version is later than your `xepersian` version and they are not in sync. In fact, you have updated part of your TeX distribution and made the harm yourself. I have no idea where you could get version `17.3` of the `xepersian` package.

**Comment by habedi** [comment]
I haven't updated anything; I've just installed Debian stable(Debian 9 AMD64) after that the makefile I used to run and build my projects wouldn't work; I only saw the error message in this issue time and time again in my makefile's output ; there is definately a bug causing the problem because it seems updating the xepersian package will solve this problem.

**Comment by habedi** [comment]
Btw, what is the next version released right after 17.2?

**Comment by vafa** [comment]
Read my previous comment. Clearly you have no single idea what you are talking about. There is no bug. Your `fontspec` version and your `xepersian` version needs to be in sync. 

`xepersian` for its internal font handling mechanism uses internal `fontspect` macros and for this reason the two packages need to be in sync. If for some reason `fontspec` changes its internal macros and `xepersian` is not updated as it uses the old internals of `fontspect`, the macros would not be defined and this is exactly what happened in your case.

Again, I have no idea what was the next version released right after 17.2. I only use and test the current version and can not afford to provide solutions/patches for users with different version. The `xepersian` package along with `bidi` package are in rapid development and there could be 20 released versions of each package every month. I hope you do not expect me to remember all the useless details.

**Comment by vafa** [comment]
I now accept donations to the xepersian package. Anyone can donate any amount to the xepersian package. If you are interested to donate to the xepersian package, then please email me at `persian-tex@tug.org` and request my bank account details.

The donations allow me to spend more time working on the xepersian package and make it even better. If you know that some other people may also be interested in donating to the xepersian package development, please let them know.
